lightningd/lightningd.c

```diff
@@ -42,6 +42,17 @@ struct peer *find_peer(struct lightningd_state *dstate, const struct pubkey *id)
 	FIXME_IMPLEMENT();
 }
 
+struct peer *find_peer_by_unique_id(struct lightningd *ld, u64 unique_id)
+{
+	struct peer *peer;
+	list_for_each(&ld->peers, peer, list)
+	{
```

Comment:
Was using `clang-format` and apparently the macro-expansion broke the format recognition. I'll fix this in a fixup.